packages/styles/scss/patterns/sub-patterns/link-list/_link-list.scss

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+@import '../../../globals/utils/hang';
+
+@mixin cta {
+  .#{$prefix}--card {
+    &__wrapper {
+      min-height: 116px;
+    }
+    &:not(:hover) {
+      background: $ui-background;
+    }
+    &__heading {
+      @include carbon--type-style('body-short-02');
+
+      color: $link-01;
+    }
+  }
+}
+
+@mixin link-list {
+  .#{$prefix}--link-list {
+    &__heading {
+      margin-top: $carbon--spacing-07;
```

Comment:
Maybe I'm missing it, but I'm not seeing this in the design specs.

Reply:
without padding looks like this. And I did not find how much margin is need from top for heading.
![image](https://user-images.githubusercontent.com/53789187/75590056-7548b380-5a4a-11ea-8648-c19521683eb5.png)

Reply:
That's fine for now.